app/controllers/webhooks_controller.rb

```diff
@@ -6,7 +6,7 @@ class WebhooksController < ::ApplicationController
   before_action :find_resource, only: %i[edit update destroy]
 
   def index
-    @webhooks = resource_base.all
+    @webhooks = resource_base.order(:id).all
```

Comment:
Just for the demo so it's not random order, I can drop this if you already changed those files in the other PR.